frontend/src/routes/Receive.tsx

```diff
@@ -16,11 +17,20 @@ function Receive() {
     const queryClient = useQueryClient()
 
     function handleContinue() {
-        navigate("/")
+        if (!amount || typeof parseInt(amount) !== "number") {
+            toast("That doesn't look right")
+            return
+          } else if (parseInt(amount) <= 0) {
+            navigate('/')
+            toast("You can't receive nothing")
+            return
+        }
+        if (amount) {
         const params = objectToSearchParams<ReceiveParams>({ amount, description })
         // Important! Otherwise we might see a stale bip21 code
         queryClient.invalidateQueries({ queryKey: ['bip21'] })
         navigate(`/receive/qr?${params}`)
+          }
```

Comment:
this formatting looks weird, should run the linter

Reply:
I just briefly went over the error handling again for the toast strings and I realized they refuse to display on the receive page. they work everywhere else but to see them on receive you must first navigate home